carma_cooperative_perception/src/multiple_object_tracker_component.cpp

```diff
@@ -452,6 +453,92 @@ static auto predict_track_states(std::vector<Track> tracks, units::time::second_
   return tracks;
 }
 
+struct SemanticDistance2dScore
```

Comment:
let's add documentation what this class is about

Reply:
Done